common/src/main/resources/META-INF/native-image/io.netty/common/native-image.properties

```diff
@@ -12,4 +12,6 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-Args = --initialize-at-run-time=io.netty.util.AbstractReferenceCounted,io.netty.util.concurrent.GlobalEventExecutor,io.netty.util.concurrent.ImmediateEventExecutor,io.netty.util.concurrent.ScheduledFutureTask,io.netty.util.internal.ThreadLocalRandom
+Args = --initialize-at-run-time=io.netty.util.AbstractReferenceCounted,io.netty.util.concurrent.GlobalEventExecutor,io.netty.util.concurrent.ImmediateEventExecutor,io.netty.util.concurrent.ScheduledFutureTask,io.netty.util.internal.ThreadLocalRandom,io.netty.util.internal.logging.Log4JLogger \
+       -Dio.netty.noUnsafe=true \
```

Comment:
why this ?

Reply:
I'm not sure the `io.netty.noUnsafe=true` is really needed. I'd remove it and let the final user to add it if the user really needs it.